beacon-chain/state/v1/BUILD.bazel

```diff
@@ -101,3 +105,18 @@ go_test(
         "@org_golang_google_protobuf//proto:go_default_library",
     ],
 )
+
+#ssz_gen_marshal(
```

Comment:
There is some fastssz bug because of which the generated file includes an unused import. I generated it once, fixed the import and commented this code.